### PR TITLE
Fixed all dead links

### PR DESCRIPTION
**Pull request by PierreT20**

### CODE REVIEW

File: src/components/CallToAction/index.test.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ describe('CallToAction Component', () => {
   it('ensures the Sign Up link points to the correct path', () => {
     render(<CallToAction />);
     const signUpLink = screen.getByRole('link', { name: /Sign Up/i });
-    expect(signUpLink).toHaveAttribute('href', '/auth/signin');
+    expect(signUpLink).toHaveAttribute('href', 'https://trophy-tracker-wsu.vercel.app/api/auth/signin?callbackUrl=https%3A%2F%2Ftrophy-tracker-wsu.vercel.app%2F');
```
Comment (ThatZiv) [inline src/components/CallToAction/index.test.tsx:22]: Will this work locally? maybe use `String.includes()` for this?

**Review by SukrutN — APPROVED** [review]
Checked links asynchronously, all links are working and clickable. Approved.

File: src/comps/Nav.tsx [hunk 2]
```diff
@@ -159,18 +159,26 @@ export default function Nav() {
         <Navbar.Link
           as={Link}
           href="/Charts"
-          active={pathname.startsWith("/Livestream")}
+          active={pathname.startsWith("/Charts")}
```
Comment (ThatZiv) [inline src/comps/Nav.tsx:162]: We should be using all lowercase for paths

File: src/comps/Nav.tsx [hunk 3]
```diff
@@ -159,18 +159,26 @@ export default function Nav() {
...
-          active={pathname.startsWith("/Livestream")}
+          active={pathname.startsWith("/Upcoming")}
         >
           Upcoming
         </Navbar.Link>
+
+        <Navbar.Link
+          as={Link}
+          href="/HowTtWorks"
+          active={pathname.startsWith("/HowTtWorks")}
```
Comment (ThatZiv) [inline src/comps/Nav.tsx:178]: Let's keep the cases consistent or use kebab case `how-it-works` for next time

**Review by ThatZiv — APPROVED** [review]
Approved

**Pull request merged**